lib/parser/objects.ts

```diff
@@ -592,6 +595,7 @@ export class OVariable extends ObjectBase implements IVariableBase {
   type: ORead[] = [];
   defaultValue?: ORead[] = [];
   lexerToken: OLexerToken;
+  shared = false;
```

Comment:
This is parsed and assigned but never used. Are there any plans/requirements for it?

Reply:
So, normal use case for shared variables it protected types. And calling them via shared_variable.method().
I implemented share variable to use this in the resolution of this pseudo class-instance thing. But then I found out the instance may also be of non-shared type for some reason...
So I removed this from elaborate again...
But shared variable is a vhdl language of variables, I think it is fine to parse and extract?